### DISCUSSION TITLE
Should Kanban be a Writingtool? -> https://www.speare.com/ YESSS!!!

### DISCUSSION THREAD

**Discussion by benytzlich** [mgmeyers/obsidian-kanban#166] [Ideas & Feature Requests]
YESSSS!
 https://www.speare.com

another tool for writing and keeping the big picture: https://gingkowriter.com/

**Comment by jokysatria** [comment]
I try to use kanban for writing too.
Supporting dash "-" prefix (even better without prefix, card based on paragraph/markdown block) will be awesome!

**Comment by Rhydderch** [comment]
Hi @mgmeyers,
Do you still think/plan of integrating features similar to https://www.speare.com/ ? (I discovered it through the kanban project roadmap https://github.com/mgmeyers/obsidian-kanban/projects/1 )

I guess this workflow is already kinda possible in Obsidian with the kanban plugin.

**Comment by stoweboyd** [comment]
I use kanban that way, already: see [Card-Based Writing in Obsidian Using Kanban - by Stowe Boyd](https://www.workings.tools/p/card-based-writing-in-obsidian-using?s=w)

**Reply by Rhydderch** [reply]
I saw your post and I think it's a great workflow. However, for longform writing, it's not enough for me. I really need to be able to have multiple hierarchies/levels, like gingowriters allow.

**Comment by stoweboyd** [comment]
Yeah, kanban's developers are working on new means that might do it.
